post-install-setup.sh

```diff
@@ -0,0 +1,328 @@
+#!/bin/bash
```

Comment:
Hello @warthog9, I`m facing issues in running this script:

1. After creating liota user:
```
$ ./post-install-setup.sh
Gives following error:
Liota: etc dir: /etc/liota
ERROR: /etc/liota already exists, and assuming things are setup already or not but automatic running will not proceed. This is a fatal error.
```

It only copies one file liota.conf but not logging.json.

2. Also, running the script with root user gives the following error:
```
 $ _LIOTA_ROOTUSER="root" ./post-install-setup.sh 
Liota: etc dir: /etc/liota
ERROR: There is a configuration file already existing at /etc/liota/liota.conf, not going to overwrite. This is a fatal error.
```

I tried deleting the "/etc/liota" and retrying above commands but it fails again. I believe there is a double check in the script one of them will have to be removed.